python-client/giskard/testing/tests/metamorphic.py

```diff
@@ -22,6 +22,8 @@
 
 
 def _predict_numeric_result(model: BaseModel, ds: Dataset, output_proba=True, classification_label=None):
+    if model.is_text_generation:
+        return model.predict(ds).raw_prediction.flatten()
```

Comment:
@kevinmessiaen is this the only line changed apart from style fixes?

Reply:
Yes it is